src/components/@shared/atoms/Modal/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { ReactElement, ReactNode } from 'react'
 import ReactModal from 'react-modal'
-import styles from './Modal.module.css'
+import styles from './index.module.css'
 
-if (process.env.NODE_ENV !== 'test') ReactModal.setAppElement('#__next')
+if (process.env.NODE_ENV !== 'test') ReactModal.setAppElement('#root')
```

Comment:
are you sure about this not breaking the modal in the app? Cause I don't see any `root` id on any markup in the production build.

Reply:
it actually worked using the old test file, thanks for the tip!